app/views/questions/_form.html.erb

```diff
@@ -25,9 +25,8 @@
   </div>
   <div class="modal-footer">
     <% if @question.nil? %>
-      <%= link_to "See Orgs", finish_quiz_path(:quiz_id => @quiz.id), :id => 'finish-quiz', :class => "btn btn-finish-quiz", :remote => true, "data-dismiss" => "modal" %>
+      <%= link_to "See Orgs", finish_quiz_path(:quiz_id => @quiz.id), :id => 'finish-quiz', :class => "btn btn-finish-quiz" %>
```

Comment:
We should try to make this a remote submission so that we do not redirect when we finish the quiz. From what I understand, the client's vision of this feature was supposed to be all in one page. But not a blocker for now, feature for next iteration 🤙